packages/wrangler/src/d1/create.tsx

```diff
@@ -19,12 +19,17 @@ export function Options(yargs: CommonYargsArgv) {
 			type: "string",
 			demandOption: true,
 		})
+		.option("experimental", {
```

Comment:
Can this have a more verbose name? The word `experimental` is pretty widely used—something like `--experimental-backend` would fit better with the precedent of e.g. `--experimental-local`

Reply:
@penalosa this is for internal testing only

Reply:
In that case an environment variable would probably fit better? `EXPERIMENTAL_D1_BACKEND` or something?

Reply:
Nah as in I don't even think we'll merge this into main for now

Reply:
Oh right, I didn't realise the base branch wasn't `main`. In that case fire away